gcp/gcp.go

```diff
@@ -1031,8 +1031,12 @@ func (p gcpProvider) getDatabases(aggregator *sqladmin.InstancesListCall) ([]*ty
 	for _, databaseInstance := range gDatabaseList.Items {
 		instanceName := databaseInstance.Name
 
-		listOperationCall := p.sqlClient.Operations.List(p.projectID, instanceName)
-		creationTimeStamp, err := getDatabaseInstanceCreationTimeStamp(listOperationCall, instanceName)
+		operationsList, err := p.sqlClient.Operations.List(p.projectID, instanceName).Do()
```

Comment:
I suggest to use `maxResults` because we don't need the entire list of operations.